doc/requirements.md

```diff
@@ -0,0 +1,34 @@
+# Requirements
+
+## Owner 
+ - don't want to store big images online
+ - moderation
+   - want to be able to delete QR codes that are bad or malicious 
+   - able to delete players 
+
+## Player
+attributes? (e.g. Username and contact information (like what contact info tho?)
+ - Do not want to have to login to personal application using username/password as device ID (SIM, phone, w/e) can identify me 
+   - BUT use log in to add additional devices(?)
+ - QR code is scanned by the camera (camera access button) -- don't record the actual code
```

Comment:
pretty sure he _does_ want us to record the code, _not_ the image. haven't read the doc tho lmao

Reply:
Yeah, I think you're right. In the updated version on the Discord server, I corrected this. Good catch tho